backend/src/main/migrations/20211004020527-create-assigned.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Assigned', {
+      taskId: {
```

Comment:
I think you need to specify that taskId and email are primary key. To show that it is a many to many relation